python/fusion_engine_client/messages/defs.py

```diff
@@ -111,26 +111,40 @@ class MessageType(IntEnum):
 
     RESERVED = 20000
 
+    @classmethod
+    def get_type_string(cls, type, include_value=True, raise_on_unrecognized=False):
+        return MessageType.static_to_string(type, include_value=include_value,
+                                            raise_on_unrecognized=raise_on_unrecognized)
 
     @classmethod
-    def get_type_string(cls, type):
+    def static_to_string(cls, value, include_value=True, raise_on_unrecognized=False):
```

Comment:
Is there a reason to make this a static method like this instead of overriding __str__ or __repr__ ?